app/controllers/admin/attachments_controller.rb

```diff
@@ -89,7 +91,7 @@ def attachable_attachments_path(attachable)
 private
 
   def get_layout
-    design_system_actions = %w[edit update new create confirm_destroy reorder]
+    design_system_actions = %w[index edit update new create confirm_destroy reorder]
```

Comment:
We'll also need to handle that the index page is not in the same release as the other pages here.